slices/util-linux.yaml

```diff
@@ -0,0 +1,157 @@
+package: util-linux
+
+essential:
+  - util-linux_copyright
+
+slices:
+  bins:
+    essential:
+      - libblkid1_libs
+      - libc6_libs
+      - libcap-ng0_libs
+      - libcrypt1_libs
+      - libmount1_libs
+      - libpam0g_libs
+      - libselinux1_libs
+      - libsmartcols1_libs
+      - libsystemd0_libs
+      - libtinfo6_libs
+      - libudev1_libs
+      - libuuid1_libs
+      - util-linux_generated
+      - util-linux_pam-profiles
+      - util-linux_su-support
+      - zlib1g_libs
+    contents:
```

Comment:
Sry I have to throw one more request here, but to follow our own practices, we should test every binary in the slice.